src/archival/utils.rs

```diff
@@ -37,8 +43,22 @@ pub async fn update_internet_archive_urls(pool: &PgPool, job_id: String, id: i32
         .bind(job_id)
         .bind(id)
         .execute(pool)
-        .await
-        .unwrap();
+        .await?;
+    Ok(())
+}
+
+pub async fn update_internet_archive_urls_with_retry_count_inc(
```

Comment:
The name of this function is overly complicated. Just go simpler:

```suggestion
pub async fn inc_archive_request_retry_count(
```

Reply:
To be honest, an even more general renaming of the project will be needed at some point.

Reply:
True, I can see some parts getting inproved. But it's better to not add more either.

Reply:
Yeah I'll rename variables 😅. 
I couldn't think of anything for the project name, and anything I thought was either not much clear, or was even dumber 😅

Reply:
Project names are definitely my weak point too...